src/app/share/types/index.ts

```diff
@@ -10,6 +10,7 @@ export type AdvancedSharedItem = SharedFolders &
     sharingId?: string;
     sharingType: 'public' | 'private';
     encryptedPassword: string | null;
+    hybridModeEnabled: boolean | false;
```

Comment:
`boolean` already includes `false`, it is redundant

Reply:
Done